### DISCUSSION TITLE
timestamp issue

### DISCUSSION THREAD

**Discussion by happymoment-s** [vesoft-inc/nebula#4295] [Q&A]
nebula 3.1.0 (nebula-java : com.vesoft:client:3.0.0)
spring boot/java

I am using timestamp in the property of vertex.
https://docs.nebula-graph.io/3.1.0/3.ngql-guide/3.data-types/4.date-and-time/#timestamp

If the vertex is stored below a specific value (ex:1638365289), it cannot be searched with a FETCH query.
<img width="204" alt="스크린샷 2022-06-02 오전 10 25 24" src="https://user-images.githubusercontent.com/6138944/171530422-1ef0a19b-8244-4dce-8838-59582b332a25.png">

When I set an edge to the corresponding vertex and execute GO query, `_EMPTY_` value is returned.
<img width="123" alt="스크린샷 2022-06-02 오전 10 27 28" src="https://user-images.githubusercontent.com/6138944/171530443-f7149e54-aa0a-46b0-af03-3e49f26d79be.png">

1. Is this normal operation? Or is it a bug?
2. A specific value that cannot be saved keeps changing (continues increasing)
(For example, if I save as 1638365289, it is not searched, but after a while, if I save it as 1638365289, the search works well)
3. When saving, do I need to handle an exception so that a specific value is not saved? (Is it possible..?)

Please check. thank you

**Comment by xiajingchun** [comment]
Could you share the schema of the tag your created? And show me the nGQL pls. :)  About how you inserted the vertex, and how you did the search?
